docs/topics/proc_add-debug-logging-to-app-vertx.adoc

```diff
@@ -0,0 +1,21 @@
+
+[#_add_debug_logging_to_application_vertx]
```

Comment:
The anchor ID here should be `[#adding-log-output-to-your-vertx-application]` (optionally add the `_{context}` suffix, but that's probably not necessary in this case).

Applies to all other anchor IDs as well.

For more information, see the [Modular Documentation Reference Guide](https://redhat-documentation.github.io/modular-docs/#anchor-and-file-names).

Reply:
Thank you. fixed all anchors to conform to MDP guidelines